packages/web-components/src/components/cbp-banner/cbp-banner.scss

```diff
@@ -0,0 +1,31 @@
+
+:root {
+  --cbp-banner-color-bg: var(--cbp-color-info-dark);
+  --cbp-banner-color-fg: var(--cbp-color-text-lightest)
+}
+cbp-banner {
+  display: flex;
+  flex-direction: row;
+
+  &[aria-color="infoBanner"]{
+    background-color: var(--cbp-banner-color-bg);
+  }
+
+  .iconContainer{
+    align-content: center;
+    padding-inline: var(--cbp-space-4x);
```

Comment:
I like that you're using CSS logical properties, but this makes more sense to use as `gap` along with the `display: flex` declaration.